crates/services/gas_price_service/src/fuel_gas_price_updater/fuel_da_source_adapter/block_committer_source.rs

```diff
@@ -0,0 +1,60 @@
+use crate::fuel_gas_price_updater::{
+    service::Result as DaGasPriceSourceResult,
+    DaGasPrice,
+    DaGasPriceSource,
+};
+use anyhow::anyhow;
+use reqwest::Url;
+use serde::{
+    Deserialize,
+    Serialize,
+};
+
+/// This struct is used to denote the block committer da gas price source,,
+/// which receives data from the block committer (only http api for now)
+pub struct BlockCommitterDaGasPriceSource {
```

Comment:
Maybe `BlockCommitterDaBlockCosts` or something like that would be better

Reply:
addressed in https://github.com/FuelLabs/fuel-core/pull/2163/commits/74fc9bf4e08cb0d4a1a5def23f31d2c4fbd31c9f#diff-d6d135691e84a45a0d07553a66a5a6db1bf760fcfdd0b626c15a6e966ca3e09cR15-R48